node/avatar-bot/index.js

```diff
@@ -26,7 +26,7 @@
 exports.helloHangoutsChat = function helloHangoutsChat(req, res) {
   if (req.method === 'GET' || !req.body.message) {
     res.send('Hello! This function is meant to be used in a Hangouts Chat ' +
```

Comment:
Is line break needed? Seems well under line length limit otherwise.

Reply:
barely over the limit at 82 cols